airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py

```diff
@@ -1,102 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-import responses
-from airbyte_cdk.models import SyncMode
-from source_pivotal_tracker.source import Activity, Epics, Labels, PivotalAuthenticator, ProjectMemberships, Projects, Releases, Stories
-
-auth = PivotalAuthenticator("goodtoken")
-project_args = {"project_ids": [98, 99], "authenticator": auth}
-stream_slice = {"project_id": 99}
-
-
-@responses.activate
-def test_projects_stream(projects_response):
```

Comment:
These tests should not be deleted. Instead, they should be aligned with the new approach

Reply:
@askarpets Could you please eloborate this process?
Low code cdk usually perform tests within standard acceptance-test yaml file isnt it?

Reply:
You can retrieve a stream needed for a test from the source instead of importing its class. Please see Gitlab [implementation](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-gitlab/unit_tests/test_streams.py)

Reply:
For this specific scenario of pivotal-tracker, the connection check and streams check are already done via standard acceptance-test-config.yaml file itself, these would be duplicate checks for this connector.
As the gitlab connector has stream slices and oauth config for additional checks, it might require the unit tests but it would be not required here, would be my suggestion

Reply:
@lazebnyi could you please take a look at the comment above and let us know if these tests should be kept?